ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Validators/TaskDTOValidator.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Validators
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using ExadelTimeTrackingSystem.BusinessLogic.DTOs;
+    using ExadelTimeTrackingSystem.BusinessLogic.Services.Abstract;
+    using ExadelTimeTrackingSystem.BusinessLogic.Validators.Abstract;
+    using FluentValidation;
+
+    public class TaskDTOValidator : BaseTaskDTOValidator<TaskDTO>
```

Comment:
Now you don't need this validator : we validate only input dtos, and now seems there are no endpoints which take TaskDTO as an input